packages/website/docs/adding-suggested-searches.md

```diff
@@ -13,6 +13,13 @@ const querySuggestionsPlugin = createQuerySuggestionsPlugin({
   searchClient,
   indexName: 'instant_search_demo_query_suggestions',
 });
+const querySuggestionsPluginWithCategories = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'instant_search_demo_query_suggestions',
+  categoryAttribute: 'instant_search.facets.exact_matches.categories.value',
```

Comment:
I'm assuming this is the `categoryAttribute` to use for this record schema:

```
{
  "instant_search": {
    "exact_nb_hits": 79,
    "facets": {
      "exact_matches": {
        "categories": [
          {
            "value": "Computers & Tablets",
            "count": 52
          },
          {
            "value": "Tablets",
            "count": 23
          }
        ],
        "hierarchicalCategories.lvl0": [
          {
            "value": "Computers & Tablets",
            "count": 52
          }
        ],
        "hierarchicalCategories.lvl1": [
          {
            "value": "Computers & Tablets > Tablets",
            "count": 23
          }
        ],
        "hierarchicalCategories.lvl2": [
          {
            "value": "Computers & Tablets > Tablets > All Tablets",
            "count": 22
          }
        ]
      },
      "analytics": {
        "categories": [],
        "hierarchicalCategories.lvl0": [],
        "hierarchicalCategories.lvl1": [],
        "hierarchicalCategories.lvl2": []
      }
    }
  },
  "nb_words": 1,
  "popularity": 556,
  "query": "amazon",
  "objectID": "amazon"
}
```
I also tried `instant_search.facets.exact_matches.categories` (without `value`) but both don't seem to be working. Could you take a look when you get a chance @francoischalifour ?

Also since adding these params I'm getting a ```createQuerySuggestionsPlugin.js?8f99:1 Uncaught (in promise) TypeError: Cannot read property 'map' of undefined``` error when typing in the autocomplete.

Thanks for your help!

Reply:
https://github.com/algolia/autocomplete/pull/454/commits/1ac9b6ba4e00dad1f5a7436d2b0749b2ec0f72e1